eclass/dune.eclass

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# @ECLASS: dune.eclass
+# @MAINTAINER:
+# maintainer-needed@gentoo.org
+# @AUTHOR:
+# Rafael Kitover <rkitover@gmail.com>
+# @SUPPORTED_EAPIS: 5 6 7
+# @BLURB: Provides functions for installing dune packages.
+# @DESCRIPTION:
+# Provides dependencies on dune and ocaml and default src_compile, src_test and
+# src_install for dune-based packages.
+
+# @ECLASS-VARIABLE: DUNE_PKG_NAME
+# @DESCRIPTION:
+# Sets the actual dune package name, if different from gentoo package name.
+# Set before inheriting the eclass.
+
+case ${EAPI:-0} in
+	5|6|7) ;;
+	*) die "${ECLASS}: EAPI ${EAPI} not supported" ;;
+esac
+
+RDEPEND=">=dev-lang/ocaml-4:="
+DEPEND="${RDEPEND}
```

Comment:
This will overwrite existing DEPEND if the ebuild inherits several eclasses, making it incompatible with other eclasses?

Reply:
This is OK. It's not easy to find, but the `DEPEND` variables get special treatment in eclasses:

https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-10600010.2

Any new eclasses should go to the dev list for review, though =)

Reply:
Posted eclass to gentoo-dev.

Reply:
Well, I got not a single response there.

Would it help if I add myself as a proxy-maint to the metadata.xml to move this forward?